entries/entry01.md

```diff
@@ -1,7 +1,18 @@
 # Entry 1
```

Comment:
Looking forward to seeing what you make! However, missing sources, EDP, and skills.